libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -876,6 +876,69 @@ describe('Date range picker', function () {
     });
   });
 
+  it('should mark select calculator picker as invalid only when the entire date range picker is invalid', fakeAsync(() => {
```

Comment:
I'm having difficulty understanding this statement. If we change it to, "should only mark the calculator select as invalid if the range is invalid" does it still communicate what the test is capturing?

Reply:
`should not mark calculator as invalid when the error is from the datepicker` ?

Reply:
That works.